docs/installation/requirements.md

```diff
@@ -4,25 +4,27 @@ This page presents an overview of the main requirements, both in terms of **reso
-As a rule of thumb, the Liqo control plane as a whole, executed on a two-node KinD cluster, peered with a remote cluster, and while offloading 100 pods, conservatively demands for less than:
+While the exact numbers depend on the **number of established peerings**, **number of offloaded pods** and on the **size of the cluster**, as a ballpark figure the entire Liqo control plane, executed on a two-nodes KinD cluster, peered with one remote cluster, and while offloading 100 pods, requires less than:
 
-* Half a CPU core (only during transient periods, while CPU consumption is practically negligible in all the other instants).
+* 0.5 CPU cores (only during transient periods, while CPU consumption is practically negligible in all the other instants).
 * 200 MB of RAM (this metric increases the more pods are offloaded to remote clusters).
 * 5 Mbps of cross-cluster control plane traffic (only during transient periods). Data plane traffic, instead, depends on the applications and their actual placements across the clusters.
 
 A thorough analysis of the Liqo performance compared to vanilla Kubernetes, including the characterization of the resources consumed by Liqo, is presented in a [dedicated blog post](https://medium.com/the-liqo-blog/benchmarking-liqo-kubernetes-multi-cluster-performance-d77942d7f67c).
 
 ## Connectivity
 
-As detailed in the [peering section](/features/peering), Liqo supports two alternative peering approaches, each characterized by **different requirements in terms of network connectivity** (i.e., mutually reachable endpoints).
-Specifically, the establishment of an [**out-of-band control plane peering**](FeaturesPeeringOutOfBandControlPlane) necessitates **three separated traffic flows** (hence, exposed endpoints), while the [**in-band control plane peering**](FeaturesPeeringInBandControlPlane) approach relaxes this requirement to a **single endpoint**, as all control plane traffic is tunneled inside the cross-cluster VPN.
+Liqo supports two alternative peering approaches, each characterized by **different requirements in terms of network connectivity** (i.e., mutually reachable endpoints):
+
+* [**Out-of-band control plane peering**](FeaturesPeeringOutOfBandControlPlane): it requires **three separated traffic flows** (hence, three exposed endpoints).
+* [**in-band control plane peering**](FeaturesPeeringInBandControlPlane): it requires a **single endpoint**, as all control plane traffic is tunneled inside the cross-cluster VPN.
+
+More details available in the [peering section](/features/peering).
 
 ```{admonition} Note
-The two peering approaches are **non-mutually exclusive**.
-In other words, a single cluster can leverage different approaches toward different remote clusters, in case all connectivity requirements are fullfilled.
+The two peering approaches are **non-mutually exclusive**: a cluster can leverage different approaches toward different remote clusters, provided that the connectivity requirements are verified.
```

Comment:
To me, *verified* conveys a different meaning compared to the original *fulfilled*. Is this intentional?